server/container/util-java8/src/test/java/org/apache/james/util/OptionalUtilsTest.java

```diff
@@ -120,6 +120,11 @@ public void orShouldReturnFirstValueWhenBothValues() {
             .contains(1);
     }
 
+    @Test
```

Comment:
There is a typo in this commit name by the way "diffirent" don't forget to fix it when you will do your final rebase